en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -36,23 +40,113 @@ often working alongside Core Committers.
+
+While anyone in the community can review a pull request, only reviews from maintainers have weight of authority - and only maintainers can meaningfully approve changes. The Peer Reviewer role ensures members of the community have that authority.
+
+#### Responsibilities {#peer-reviewer-responsibilities}
+
+* Review pull requests, ensuring documented processes and best practices are being followed
+  * Refer to [how to review pull requests](/contributing/triage_resources/#how-to-review-pull-requests) and [contributing code](/contributing/code/)
+* Escalate pull requests to Core Committers if the PR is very complex and you don't feel confident reviewing it
+  * If another reviewer feels confident, they can offer to review it instead
+* Escalate pull requests to Core Committers if the contributor is rude/argumentative/hard to deal with
+  * If you feel confident doing so, politely request the contributor behave in accordance with our [code of conduct](./code_of_conduct), and close the pull request if they refuse to do so
+* After two maintainers have approved the pull request, merge it
+  * The person who merges the PR must also understand and approve the changes
+  * The person who contributed the PR must not be included as one of the two approving reviewers
+* If you don't feel confident clicking merge, ask for a second opinion. Don’t approve the pull request if you wouldn’t feel confident merging it - instead, say where you got to with it (e.g. tested, worked well locally, couldn’t find any problems but want another opinion) and get another reviewer to check it
+  * If no other reviewer feels confident about it, but it seems like it’s probably a good change in general, escalate to Core Committers
+
+In addition to the above responsibilities, if a reviewer doesn’t review any pull requests in a 12 month period, a member of the Core Committers or the CMS Squad team leader will reach out and ask if they want to continue being a reviewer. If they don’t respond within a month (or they respond saying they don’t want to be a reviewer anymore), their permissions will be revoked and they will be removed from the team.
+
+If the reviewer fails to contribute for 24 months in a row, they will be removed from the team until such a time as they can commit to performing the responsibilities of the role.
+
+#### Onboarding new peer reviewers
+
+Members of the community who have shown an interest in helping to maintain Silverstripe CMS, who have shown themselves to be trustworthy, who follow the code of conduct, and who have provided good contributions may be invited by a Core Committer or the CMS Squad team leader to join the peer review team.
```

Comment:
Done, except I moved the"may be invited by a Core Committer or the CMS Squad team leader to join the peer review team" part to be before the list instead of dangling on the last bullet point.